src/applications/_mock-form/helpers.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const hasDirectDeposit = formData =>
+  formData.viewNoDirectDeposit !== true;
+
+export const directDepositWarning = (
+  <div className="pension-dd-warning">
```

Comment:
This seems like it might be misleading or I'm just confused. The class id defined in `src/applications/pensions/sass/pensions.scss`, but it doesn't look like _that_ sass file is imported into the sass for this mock form app.

https://github.com/department-of-veterans-affairs/vets-website/blob/7a4797f81610fce68477f46f36e31b2f281da655/src/applications/pensions/sass/pensions.scss#L38-L40

Also, if we do need that styling it looks like a utility class would work.

Reply:
When I switched to using the `directDeposit` definition, this file ended up being removed.